tests/system/src/test/java/test/robot/javafx/scene/SRGBTest.java

```diff
@@ -217,7 +245,9 @@ public void screenCaptureTest() {
     // colors but they might both be working in the wrong space. We use an
     // AWT Robot to verify that they are working in sRGB.
     // Timeout for potential hang on XWayland, see JDK-8335468.
-    @Test(timeout = 15000)
+    // the same timeout will apply to the rest of the tests
+    // @Test(timeout = 15000)
```

Comment:
I would remove this commented out line, since it is a JUnit 4 holdover. Also, would it be better to apply the `@Timeout` annotation to just this method? Either is fine with me.

Reply:
good point